src/components/BookList.js

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { loadBooks } from '../redux/books/books';
+import Book from './Book';
+
+const List = () => {
+  const dispatch = useDispatch();
+  const books = useSelector((state) => state.booksReducer);
+
+  useEffect(() => {
+    dispatch(loadBooks());
+  }, []);
+
+  return (
+    <ul id="bookList">
+      {books.map((book) => (
+        <li className="book" key={book.id}>
+          <Book
+            title={book.name}
+            category={book.category}
+            author={book.author}
+            progress={book.progress}
+            currentChapter={book.currentChapter}
+            id={book.id}
+          />
```

Comment:
- [OPTIONAL] Please, be careful when giving undefined keys to your Book component like `book.author` because the console log is showing this message. 😉 

![author-msg](https://user-images.githubusercontent.com/79876959/141389512-3a541011-7626-43f3-99fe-12cf21522548.PNG)

Reply:
I'm sorry for that mistake. I'll fix this issue soon!